### DISCUSSION TITLE
add iso date format in the display section of date/time fields

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13979] [Feature Requests]
year-month-day hour:min:seconds is a popular format, and you already have long/short datetime formats, so ISO would round off the available date time formats without having to resort to custom formatting.
![Screenshot from 2022-06-18 18-29-17](https://user-images.githubusercontent.com/95102939/174449923-f583aa46-5f04-41f0-a7aa-528fdb04ceb5.png)

**Comment by benhaynes** [comment]
I agree we should have more options... and should also give examples of each (`Long, eg: July 5, 2020`). I know these are dynamic and get translated, but examples would help. Also, we could make the example based on the current user's language.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
